src/Aspire.Dashboard/Authentication/WebAppAuthenticationHandler.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Claims;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.Extensions.Options;
+
+namespace Aspire.Dashboard.Authentication;
+
+public sealed class WebAppAuthenticationHandler(
```

Comment:
I've been thinking about what to call the non-OTLP endpoint. I like Frontend. It's very clear what the difference is between frontend and OTLP.

Also, for consistency with the Otlp auth that does this, it should have composite in the name.

So, `FrontendCompositeAuthenticationHandler`.

Reply:
I pushed an update that renames "web app" to "frontend" throughout.